wled00/xml.cpp

```diff
@@ -637,7 +642,7 @@ void getSettingsJS(byte subPage, Print& settingsScript)
     #if defined(ARDUINO_ARCH_ESP32)
       ESP.getChipModel(),
     #else
-      F("esp8266"),
+      "esp8266",
```

Comment:
IMO we should be going the other way around on ESP8266 -- better to waste ROM with multiple copies than even a little RAM.

...I should really look in to a better solution; I'm 75% confident that there's some linker trickery that could be done to make a version of `F("string")` that also coalesces duplicates.

Reply:
"esp8266" is actually used elsewhere in other libraries. As it is not stored in PROGMEM linker reuses it and only one instance is used throughout.

Reply:
Are you ok to approve as it stands or can you update so we can resolve this issue @willmmiles ?

Reply:
Oh sorry, I thought I'd flagged this as resolved already!  I'm happy to accept as is and approach string RAM savings later in a global review or with better tools.